src/Layout/Directory.Build.props

```diff
@@ -54,6 +54,12 @@
     <BundleRuntimePacks Condition="'$(BundleRuntimePacks)' == '' and '$(DotNetBuildSourceOnly)' == 'true'">true</BundleRuntimePacks>
     <BundleNativeAotCompiler Condition="'$(BundleNativeAotCompiler)' == '' and '$(DotNetBuildSourceOnly)' == 'true' and '$(DotNetBuildUseMonoRuntime)' != 'true'">true</BundleNativeAotCompiler>
 
+    <!-- Crossgen2 is not bundled by default on platforms where Microsoft provides a package on nuget.org, 
+         because it is large (100MB+). -->
+    <_IsCommunityPlatform Condition="'$(OSName)' != 'win' and '$(OSName)' != 'osx' 
+      and !('$(OSName)' == 'linux' and ('$(TargetArchitecture)' == 'x64' or '$(TargetArchitecture)' == 'x86' or '$(TargetArchitecture)' == 'arm' or '$(TargetArchitecture)' == 'arm64'))">true</_IsCommunityPlatform>
+    <BundleCrossgen2 Condition="'$(BundleCrossgen2)' == '' and '$(DotNetBuildSourceOnly)' == 'true' and '$(DotNetBuildUseMonoRuntime)' != 'true' and '$( _IsCommunityPlatform )' == 'true'">true</BundleCrossgen2>
```

Comment:
Who's going to maintain that list? Basically if we add a new platform to the msft build how do we make sure that this information is up-to-date?

Reply:
When that happens (once in years), there are several places which distinguish official and community platforms which need to be updated, so it will be part of that work.